test/AWS.MSK.Auth.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Why would you have a test project target `netstandard2.0` since that `netstandard2.0` is not a runtime by an API contract? A test project is basically an executable that requires a runtime. I would expect this to be targeting at least .NET 8 as the latest LTS. Ideally use multi targeting to test the runtimes team supports with this library.